test_crates/src/test_cases/struct_repr_transparent_removed.rs

```diff
@@ -114,6 +114,18 @@ pub struct WithGeneric {
     _marker: std::marker::PhantomData<&'static usize>,
 }
 
+#[cfg(not(feature = "struct_repr_transparent_removed"))]
+#[repr(transparent)]
+pub struct BothTransparent {
+    pub bar: usize,
+}
+
+#[cfg(feature = "struct_repr_transparent_removed")]
+#[repr(transparent, )]
```

Comment:
Nice! Great idea.

Reply:
Actually, both `cargo fmt` and `cargo clippy` allow for this and do not enforce a change, what is a bit surprising.

I'm not sure if this could be ignored or if we should do something about it, so I will create an issue about this.